sql/core/src/main/scala/org/apache/spark/sql/execution/command/commands.scala

```diff
@@ -237,15 +237,18 @@ case class ExplainCommand(
     logicalPlan: LogicalPlan,
     override val output: Seq[Attribute] =
       Seq(AttributeReference("plan", StringType, nullable = true)()),
-    extended: Boolean = false)
+    extended: Boolean = false,
+    codegen: Boolean = false)
```

Comment:
Oh, do you mean `codegen select ...` instead `explain codegen select ...`?
I thought `codegen` was an boolean option flag, but new command is also possible if it need more other options.
What kind of usage for `codegen` command do you think? Could you enumerate some for me?